src/InjectionContainerInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\View;
+
+interface InjectionContainerInterface
```

Comment:
```suggestion
use Psr\Container\ContainerInterface;

interface InjectionContainerInterface extends ContainerInterface
```

Maybe use PSR container interface with return type `object`?

Reply:
1) We not need method `has()`.
2) Custom implementation without PSR container easer make with current interface.